contracts/kudos/README.md

```diff
@@ -0,0 +1,189 @@
+# Kudos
+
+Kudos contract for NDC
+
+## Setup [Testnet]
+
+- Init contract
+
+  ```sh
+  near call $CONRTACT_ID init '{"iah_registry": "registry-unstable.i-am-human.testnet"}' --accountId rubycoptest.testnet
```

Comment:
lets add change it to YOUR_ACCOUNT.testnet instead of putting someones account here